### ISSUE TITLE
Ruby 3.0.1 support

### ISSUE THREAD

**Issue by dvandersluis** [pry/pry-doc#113]
I'm getting the following when running pry with `pry-doc` on ruby 3.0.1:

> ruby/3.0.1 isn't supported by this pry-doc version

**Comment by therealrodk** [comment]
+1 Having the same problem.

**Comment by TylerRick** [comment]
Same thing for 3.0.2:
```
ruby/3.0.2 isn't supported by this pry-doc version
```

**Comment by Kaisen-san** [comment]
Same for 3.0.0:
```
ruby/3.0.0 isn't supported by this pry-doc version
```

I believe `pry-doc` isn't yet supporting Ruby v3 in general.

**Comment by kyrylo** [comment]
I just published Pry Doc v1.2.0: https://rubygems.org/gems/pry-doc/versions/1.2.0

It should add Ruby 3 support. Big thanks to @andrehjr who made this possible! 🎉
